packages/cysync-core/src/components/GenericLoader.tsx

```diff
@@ -0,0 +1,14 @@
+import { DialogBox, Image, loaderGrayIcon } from '@cypherock/cysync-ui';
+import React from 'react';
+
+export const GenericLoader: React.FC = () => (
```

Comment:
```suggestion
export const LoaderDialog: React.FC = () => (
```

Reply:
Let's do this rename in a separate PR, as this is used by many other dialogs